documentation/config_file_auth.markdown

```diff
@@ -176,6 +177,9 @@ If a request matches the rule, Puppet Server checks the request's authenticated
     -   A `certname` key equivalent to a bare string.
 
     If the request's authenticated name matches the parameter's value, Puppet Server allows it.
+
+> **Note:** If you are using Puppet Server with the CA disabled, you must use OID values for the extensions. Puppet Server will not be able to resolve [short names][] in this mode.
```

Comment:
Would it make more sense to link from `OID values for the extensions` rather than `short names`? I expect a link to take me to a page about whatever the link says, ya know?

Reply:
I guess I wanted to link "short names" because they're kind of a puppet-specific concept, where as OID values are common to OpenSSL and certs in general (in fact our docs link you out to a wikipedia page). There isn't an actual linkable section that talks about them that I could find, however. Do you think linking to the [table](https://puppet.com/docs/puppet/6.17/ssl_attributes_extensions.html#puppet_registered_ids) is better?

Reply:
ah, that makes sense. yeah, I think the table is clearer for this.